src/app/electra/epics.ts

```diff
@@ -55,3 +55,26 @@ export function generateHDWallet(action$ : any , store: Store<any>) {
     })
   })
 }
+
+export function getGlobalBalance(action$: any, store: Store<any>) {
+  return action$.ofType(ActionNames.GET_GLOBAL_BALANCE)
+  .map(() => store.getState().electraReducer.electraJs)
+  .filter((electraJs: any) => electraJs)
+  .map((electraJs: any) => electraJs.webServices.getBalanceFor())
```

Comment:
I think you are not using the appropriate api.
You are supposed to use `electraJs.wallet.getBalance()`